src/main/steps/respondent/miam/miam-start/content.test.ts

```diff
@@ -32,10 +32,10 @@ const cyContent = {
   miamSubFields: miam_collapse_content_cy,
   threeHint: 'This is a 8 character code',
   summaryText: 'Contacts for help',
-  continue: 'Save and continue',
+  onlyContinue: 'Save and continue',
```

Comment:
Should this be only 'Continue'?

Reply:
this is a label, we can have any value, but for button we need to have some thing like below:
  onlyContinue: {
    text: l => l.onlyContinue,
  },